app/assets/stylesheets/app/facility_list.scss

```diff
@@ -0,0 +1,49 @@
+.facility_list_container.tile{
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: left; 
+}
+
+.facility_listing.tile{
```

Comment:
We standardly put a space before the curly brace. Can you adjust this file to match that style?